docker/Dockerfile

```diff
@@ -11,10 +11,11 @@ RUN apt-get update && apt-get install -y \
 COPY requirements.txt ./
 
 RUN set -x && pip install --no-cache-dir -r requirements.txt
-RUN set -x && pip install gunicorn
+RUN set -x && pip install ujson gunicorn==19.9.0
```

Comment:
I used `ujson` in the past and I would be careful with it. It has not been properly maintained since 2017 and it is prone to some segfaults instead of raising exceptions properly in some cases. If you are parsing json that you generated, I think it is fine, but I would not rely on it for user provided json.
https://github.com/esnme/ultrajson

Reply:
Second this. I believe we only use it for serializing, not deserializing though.